libflux/src/flux/semantic/builtins.rs

```diff
@@ -288,6 +288,42 @@ pub fn builtins() -> Builtins<'static> {
                 "from" => Node::Builtin("forall [t0] (driverName: string, dataSourceName: string, query: string) -> [t0]"),
                 "to" => Node::Builtin("forall [t0] (<-tables: [t0], driverName: string, dataSourceName: string, table: string, ?batchSize: int) -> [t0]"),
             }),
+            "strings" => Node::Package(maplit::hashmap! {
+                "title" => Node::Builtin("forall [] (s: string) -> string"),
```

Comment:
I was expecting most of these functions to have a `v` parameter:

    forall [] (v: string) -> string

https://github.com/influxdata/flux/blob/1560758c33cf0e8ef2db3e7659480ad65e35c51e/stdlib/strings/strings.go#L35

Reply:
Ah, yeah, fixed that. Must have gotten tunnel vision looking at the function definitions from the go functions. Thanks.